b_pull_Landsat_SRST_poi/py/runGEEperTile.py

```diff
@@ -42,45 +40,45 @@
 # get extent info
 extent = yml["extent"][0]
 
-locations = read_feather("b_pull_Landsat_SRST_poi/out/locations_with_WRS2_pathrows_latlon.feather")
+# get current tile
+with open("b_pull_Landsat_SRST_poi/out/current_tile.txt", "r") as file:
+  tiles = file.read()
 
+# read in locations file
+locations = read_feather("b_pull_Landsat_SRST_poi/out/locations_with_WRS2_pathrows_latlon.feather")
+# and subset for the current tile
 locations_subset = locations.query( "`WRS2_PR` == @tiles")
 
 ##############################################
 ##---- CREATING EE FEATURECOLLECTIONS   ----##
 ##############################################
 
-
-wrs = (ee.FeatureCollection("projects/ee-ls-c2-srst/assets/WRS2_descending")
-  .filterMetadata("PR", "equals", tiles))
-
 # store path and row for subsetting the stacks so there is not overlap between PR pulls
 w_p = int(str(tiles)[0:3])
 w_r = int(str(tiles)[3:6])
 
 #grab images and apply scaling factors
 l7 = (ee.ImageCollection("LANDSAT/LE07/C02/T1_L2")
-    .map(apply_scale_factors)
```

Comment:
Note that this is moved to the end of filtering the stack to reduce processing time.